src/api/performanceApi.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+import humps from 'humps';
+import { post } from './ApiUtils';
+
+axios.defaults.headers.post['Access-Control-Allow-Origin'] = '*';
+
+const createPerformance = async (collectionName, data) => {
```

Comment:
we can probably do `export const createPerformance.....` 

instead of `export default createPerformance` since we will likely add new apis.